### PR TITLE
Logging cleanup

### PR DESCRIPTION
**Pull request by joshmoore**
Migrates nearly all print statements (other than one in `download`)
to `logging` calls. `info` now defaults to INFO and `download` to
DEBUG.

The S3/HTTP connection pool logging is still an issue making any
DEBUG level configuration difficult to read.

### CODE REVIEW

**Comment by joshmoore** [comment]
e.g. 

```
(demo) /opt/ome-zarr-py $ome_zarr info https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr
INFO:ome_zarr:root_attr: multiscales
INFO:ome_zarr:root_attr: omero
INFO:ome_zarr:resolution: 0
INFO:ome_zarr: - shape (t, c, z, y, x) = (1, 2, 236, 275, 271)
INFO:ome_zarr: - chunks =  ['1', '1', '1', '275', '271']
INFO:ome_zarr: - dtype = >u2
INFO:ome_zarr:resolution: 1
INFO:ome_zarr: - shape (t, c, z, y, x) = (1, 2, 236, 137, 135)
INFO:ome_zarr: - chunks =  ['1', '1', '1', '137', '135']
INFO:ome_zarr: - dtype = >u2
```

vs

```
(demo) /opt/ome-zarr-py $ome_zarr -v info https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr 2>&1 | grep -v urllib3
INFO:ome_zarr:root_attr: multiscales
DEBUG:ome_zarr:[{'datasets': [{'path': '0'}, {'path': '1'}], 'version': '0.1'}]
INFO:ome_zarr:root_attr: omero
DEBUG:ome_zarr:{'channels': [{'active': True, 'coefficient': 1.0, 'color': '0000FF', 'family': 'linear', 'inverted': False, 'label': 'LaminB1', 'window': {'end': 1500.0, 'max': 65535.0, 'min': 0.0, 'start': 0.0}}, {'active': True, 'coefficient': 1.0, 'color': 'FFFF00', 'family': 'linear', 'inverted': False, 'label': 'Dapi', 'window': {'end': 1500.0, 'max': 65535.0, 'min': 0.0, 'start': 0.0}}], 'id': 6001240, 'name': 'B1_C1.tif', 'rdefs': {'defaultT': 0, 'defaultZ': 118, 'model': 'color'}, 'version': '0.1'}
INFO:ome_zarr:resolution: 0
INFO:ome_zarr: - shape (t, c, z, y, x) = (1, 2, 236, 275, 271)
INFO:ome_zarr: - chunks =  ['1', '1', '1', '275', '271']
INFO:ome_zarr: - dtype = >u2
INFO:ome_zarr:resolution: 1
INFO:ome_zarr: - shape (t, c, z, y, x) = (1, 2, 236, 137, 135)
INFO:ome_zarr: - chunks =  ['1', '1', '1', '137', '135']
INFO:ome_zarr: - dtype = >u2
DEBUG:ome_zarr:[([dask.array<from-zarr, shape=(1, 2, 236, 275, 271), dtype=>u2, chunksize=(1, 1, 1, 275, 271), chunktype=numpy.ndarray>, dask.array<from-zarr, shape=(1, 2, 236, 137, 135), dtype=>u2, chunksize=(1, 1, 1, 137, 135), chunktype=numpy.ndarray>], {'channel_axis': 1, 'colormap': [<vispy.color.colormap.Colormap object at 0x7fd6f8d7b438>, <vispy.color.colormap.Colormap object at 0x7fd6f8d7b4a8>], 'contrast_limits': [[0.0, 1500.0], [0.0, 1500.0]], 'name': ['LaminB1', 'Dapi'], 'visible': [True, True]}), (dask.array<getitem, shape=(1, 236, 275, 271), dtype=int64, chunksize=(1, 1, 275, 271), chunktype=numpy.ndarray>, {'name': '0', 'color': {1: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 10: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 11: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 12: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 13: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 14: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 15: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 16: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 17: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 18: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 19: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 2: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 20: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 21: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 22: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 23: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 24: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 25: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 26: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 27: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 28: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 29: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 3: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 30: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 31: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 32: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 33: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 34: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 35: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 36: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 37: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 38: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 39: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 4: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 40: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 41: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 42: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 43: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 44: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 45: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 46: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 47: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 48: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 49: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 5: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 50: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 51: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 52: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 53: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 54: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 55: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 56: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 57: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 58: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 59: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 6: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 60: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 61: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 7: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 8: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255], 9: [0.5019607843137255, 0.5019607843137255, 0.5019607843137255, 0.5019607843137255]}}, 'labels')]
```

As well as:

```
ome_zarr download https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr
downloading to 6001240.zarr
[###                                     ] | 9% Completed |  2.6s

```

v.

```
(demo) /tmp $ome_zarr -v download https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr
INFO:ome_zarr:image_id 6001240
INFO:ome_zarr:datasets ['0', '1']
downloading to 6001240.zarr
INFO:ome_zarr:resolution 1...
[######                                  ] | 16% Completed |  3.6s
```

File: ome_zarr.py [hunk 1]
```diff
@@ -241,16 +252,10 @@ def load_ome_zarr(self):
...
-                resolution,
-                "shape (t, c, z, y, x)",
-                data.shape,
-                "chunks",
-                chunk_sizes,
-                "dtype",
-                data.dtype,
-            )
+            LOGGER.info("resolution: %s", resolution)
+            LOGGER.info(" - shape (t, c, z, y, x) = %s", data.shape)
```
Comment (manics) [inline ome_zarr.py:256]: What do you think about making this into a single log statement containing `\n`? If in future these logs are automatically parsed it'll be easier.

**Comment by joshmoore** [comment]
@manics : test of the log output now green again.

File: ome_zarr.py (hunk 1 above)
Comment (joshmoore) [reply ome_zarr.py:256]: I personally find the indentation then harder to read. I assume if we ever want these parsed we'll use a json or yaml output, no?

Comment (manics) [reply ome_zarr.py:256]: If our other projects are anything to go by we'll just end up parsing the unmodified output 😀. Merging anyway.

**Pull request merged**